### ISSUE TITLE
training failed

### ISSUE THREAD

**Issue by Duffyxx** [kozistr/rcan-tensorflow#5]
Hello, I use your code to train the model using  DIV2K, but can not get the training result, the output of the network is black image, my training loss is like:
```
[+] 28 epochs 23000 steps loss : 148.37326050 PSNR : -43.0488 SSIM : -0.0124
[+] 28 epochs 23100 steps loss : 157.56805420 PSNR : -44.0248 SSIM : -0.0029
[+] 29 epochs 23200 steps loss : 150.22140503 PSNR : -43.8336 SSIM : -0.0069
[+] 29 epochs 23300 steps loss : 131.76765442 PSNR : -42.0612 SSIM : 0.0020
[+] 29 epochs 23400 steps loss : 137.01979065 PSNR : -42.6417 SSIM : -0.0033
[+] 29 epochs 23500 steps loss : 139.48718262 PSNR : -42.9347 SSIM : -0.0040
[+] 29 epochs 23600 steps loss : 140.33860779 PSNR : -43.1099 SSIM : -0.0073
[+] 29 epochs 23700 steps loss : 137.87838745 PSNR : -43.3312 SSIM : -0.0022
[+] 29 epochs 23800 steps loss : 145.82002258 PSNR : -43.3227 SSIM : -0.0037
[+] 29 epochs 23900 steps loss : 154.25614929 PSNR : -44.0416 SSIM : -0.0016
[+] 30 epochs 24000 steps loss : 149.05319214 PSNR : -43.9319 SSIM : -0.0016
[+] 30 epochs 24100 steps loss : 154.17941284 PSNR : -44.0751 SSIM : -0.0096
[+] 30 epochs 24200 steps loss : 137.07835388 PSNR : -43.2535 SSIM : -0.0014
[+] 30 epochs 24300 steps loss : 156.63449097 PSNR : -44.3344 SSIM : 0.0001
[+] 30 epochs 24400 steps loss : 135.80091858 PSNR : -42.8577 SSIM : -0.0162
[+] 30 epochs 24500 steps loss : 138.67384338 PSNR : -43.0114 SSIM : 0.0011
[+] 30 epochs 24600 steps loss : 151.06549072 PSNR : -43.6310 SSIM : -0.0081
[+] 30 epochs 24700 steps loss : 158.61445618 PSNR : -44.2536 SSIM : -0.0010
[+] 31 epochs 24800 steps loss : 144.03248596 PSNR : -43.3128 SSIM : 0.0001
```

**Comment by kozistr** [comment]
hello! thx for reporting!

followed as #3, I also failed to reproduce the performance and it just diverged :(
I just referenced the official implementation (pytorch code), the paper and ported into tensorflow code.
Maybe there're some issues w/ pre-processing in my code!

so sorry for that I will try to fix that issue!

**Comment by weizhiliang0520** [comment]
> Hello, I use your code to train the model using DIV2K, but can not get the training result, the output of the network is black image, my training loss is like:
> 
> ```
> [+] 28 epochs 23000 steps loss : 148.37326050 PSNR : -43.0488 SSIM : -0.0124
> [+] 28 epochs 23100 steps loss : 157.56805420 PSNR : -44.0248 SSIM : -0.0029
> [+] 29 epochs 23200 steps loss : 150.22140503 PSNR : -43.8336 SSIM : -0.0069
> [+] 29 epochs 23300 steps loss : 131.76765442 PSNR : -42.0612 SSIM : 0.0020
> [+] 29 epochs 23400 steps loss : 137.01979065 PSNR : -42.6417 SSIM : -0.0033
> [+] 29 epochs 23500 steps loss : 139.48718262 PSNR : -42.9347 SSIM : -0.0040
> [+] 29 epochs 23600 steps loss : 140.33860779 PSNR : -43.1099 SSIM : -0.0073
> [+] 29 epochs 23700 steps loss : 137.87838745 PSNR : -43.3312 SSIM : -0.0022
> [+] 29 epochs 23800 steps loss : 145.82002258 PSNR : -43.3227 SSIM : -0.0037
> [+] 29 epochs 23900 steps loss : 154.25614929 PSNR : -44.0416 SSIM : -0.0016
> [+] 30 epochs 24000 steps loss : 149.05319214 PSNR : -43.9319 SSIM : -0.0016
> [+] 30 epochs 24100 steps loss : 154.17941284 PSNR : -44.0751 SSIM : -0.0096
> [+] 30 epochs 24200 steps loss : 137.07835388 PSNR : -43.2535 SSIM : -0.0014
> [+] 30 epochs 24300 steps loss : 156.63449097 PSNR : -44.3344 SSIM : 0.0001
> [+] 30 epochs 24400 steps loss : 135.80091858 PSNR : -42.8577 SSIM : -0.0162
> [+] 30 epochs 24500 steps loss : 138.67384338 PSNR : -43.0114 SSIM : 0.0011
> [+] 30 epochs 24600 steps loss : 151.06549072 PSNR : -43.6310 SSIM : -0.0081
> [+] 30 epochs 24700 steps loss : 158.61445618 PSNR : -44.2536 SSIM : -0.0010
> [+] 31 epochs 24800 steps loss : 144.03248596 PSNR : -43.3128 SSIM : 0.0001
> ```

Me too
`[-] No checkpoint file found
2021-05-10 03:05:27.163387: W tensorflow/core/framework/cpu_allocator_impl.cc:81] Allocation of 37748736 exceeds 10% of system memory.
2021-05-10 03:05:27.195992: W tensorflow/core/framework/cpu_allocator_impl.cc:81] Allocation of 18874368 exceeds 10% of system memory.
2021-05-10 03:05:27.206843: W tensorflow/core/framework/cpu_allocator_impl.cc:81] Allocation of 18874368 exceeds 10% of system memory.
2021-05-10 03:05:27.216624: W tensorflow/core/framework/cpu_allocator_impl.cc:81] Allocation of 37748736 exceeds 10% of system memory.
2021-05-10 03:05:27.225881: W tensorflow/core/framework/cpu_allocator_impl.cc:81] Allocation of 150994944 exceeds 10% of system memory.
[+] 0 epochs 0 steps loss : 127.28887939 PSNR : -42.8687 SSIM : 0.0001`
